includes/class-wc-payments-utils.php

```diff
@@ -583,4 +583,37 @@ public static function get_cached_minimum_amount( $currency ) {
 		$cached = get_transient( 'wcpay_minimum_amount_' . strtolower( $currency ) );
 		return (int) $cached ? (int) $cached : null;
 	}
+
+	/**
+	 * Locks an order for payment intent processing for 5 minutes.
+	 *
+	 * @param WC_Order $order  The order that is being paid.
+	 * @param string   $intent_id The id of the intent that is being processed.
+	 * @return bool            A flag that indicates whether the order is already locked.
```

Comment:
Remove spacing or align descriptions.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/3520/commits/d4ee61156db4327ce654916c7d7f75800984d07c